lbry/lbry/stream/descriptor.py

```diff
@@ -14,6 +15,17 @@
 
 log = logging.getLogger(__name__)
 
+_exprs = [
+    '[<>:"/\\\|\?\*]+',  # Illegal characters
+    '[\\x00-\\x1F]+',  # All characters in range 0-31
+    '[ \t]*(\.)+[ \t]*$',  # Dots at the end
+    '(^[ \t]+|[ \t]+$)',  # Leading and trailing whitespace
+    '^CON$', '^PRN$', '^AUX$',  # Illegal names
+    '^NUL$', '^COM[1-9]$', '^LPT[1-9]$'
+]
+
+RES = re.compile('(' + '|'.join((expr for expr in _exprs)) + ')')
+
```

Comment:
Can you embed `_exprs` into the `re.compile()` expression? Also please rename `RES` to something that makes sense. General convention in Python is to use concise but descriptive variable names; especially for public (not internal to a function) variables.